transform/views/main.py

```diff
@@ -91,6 +93,16 @@ def render_pck(batch_number=False):
         logger.info("PCK:SUCCESS")
         return transformer.create_pck(transformed_data)
 
+    if survey_id in cora_surveys:
+        logger.info("PCK:CORA survey detected")
+        if batch_number:
+            return client_error("PCK:CORA surveys don't support batch numbers")
+
+        transformer = UKISTransformer(response)
+        transformed_data = transformer.transform()
+        logger.info("PCK:SUCCESS")
```

Comment:
Do these loggers have transaction id etc context? Otherwise they're probably impossible to follow in logs?

Reply:
They don't, but these endpoints are helper endpoints more than anything.  I'll stick some extra context around them though because we should make an effort to give context to all the log lines where possible.

Reply:
Should be fixed now